packages/cli/src/commands/deviceprofiles/create.ts

```diff
@@ -1,30 +1,77 @@
-import { DeviceProfile, DeviceProfileRequest } from '@smartthings/core-sdk'
+import {
+	DeviceProfile,
+	DeviceProfileRequest,
+	PresentationDeviceConfigCreate,
+	SmartThingsClient,
+} from '@smartthings/core-sdk'
 
 import { InputOutputAPICommand } from '@smartthings/cli-lib'
-
 import { buildTableOutput } from '../deviceprofiles'
+import {DeviceDefinitionRequest} from './view'
+import inquirer from 'inquirer'
 
 
-export default class DeviceProfileCreateCommand extends InputOutputAPICommand<DeviceProfileRequest, DeviceProfile> {
-	static description = 'create a new device profile'
+const capabilityBlacklist = ['healthCheck', 'execute']
```

Comment:
We should use `Blocklist` or any [other alternatives](https://thenextweb.com/dd/2020/07/13/linux-kernel-will-no-longer-use-terms-blacklist-and-slave/#:~:text=Alternatives%20for%20blacklist%2Fwhitelist%20are,the%20right%20way%20to%20go.) floating around.

Reply:
fixed